docs/source/Regions2D_functionality.ipynb

```diff
@@ -0,0 +1,2342 @@
+{
```

Comment:
<p>You say hake region but select "Trawl"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>